### DISCUSSION TITLE
Nested Modals

### DISCUSSION THREAD

**Discussion by adityaa-codes** [filamentphp/filament#6052] [Help]
Hi i have a  create form page in which i have added, createOptionsForm in some Select fields , on clicking on them it opens modal to create dependent Data,  
On Opening Model, i have a field to add media using Curator to the dependent data, is there any way i can open nested modals.
Currently its not working, its just give me blank page
Create Data Page
![image](https://user-images.githubusercontent.com/26678129/228534644-d09c097b-c193-422e-8c03-fa877b0957be.png)
Create Dependent Modal Page
![image](https://user-images.githubusercontent.com/26678129/228534718-3e97a3ac-1da9-4c14-b610-2b9fda492358.png)
On Clicking On Add Media
![image](https://user-images.githubusercontent.com/26678129/228534913-d0800365-9c67-4e79-a2e8-bc54afe22e18.png)

**Comment by danharrin** [comment]
I think you'd need to submit this as an issue on the Curator plugin, not here

**Reply by adityaa-codes** [reply]
Thank you, i'll do that
